onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4249,16 +4260,18 @@ def aten_sinh(self):
     return op.Sinh(self)
 
 
+@torch_op("aten::slice")
 def aten_slice(
     self: TensorType,
     dim: int = 0,
-    start: Optional[INT64] = None,
-    end: Optional[INT64] = None,
+    start: INT64 = None,
+    end: INT64 = None,
     step: INT64 = 1,
 ) -> TensorType:
     # slice.Tensor(Tensor(a) self, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)
 
-    raise NotImplementedError()
+    return op.Slice(self, start, end, dim, step)
```

Comment:
Consider using SequenceConstruct for start and end?

Reply:
temporarily delete this implementation, move to a new PR.